src/SConscript

```diff
@@ -94,19 +94,12 @@ def build_tests():
         env.Append(LIBS = 'gmock')
 
         if build.platform_is_windows:
-                # Subsystem change temporarily disabled due to LP #1453782.
                 # TODO(rryan): Build Mixxx core as a shared object and link it
                 # into mixxx and mixxx-test. We could build both in different
                 # environments right now but then automoc and protoc get run in
                 # both environments which makes SCons unhappy.
-                # # We want a terminal for tests.
-                # if build.machine_is_64bit:
-                #     test_env['LINKFLAGS'].remove('/subsystem:windows,5.02')
-                #     test_env['LINKFLAGS'].append('/subsystem:console,5.02')
-                # else:
-                #     test_env['LINKFLAGS'].remove('/subsystem:windows,5.01')
-                #     test_env['LINKFLAGS'].append('/subsystem:console,5.01')
-
+                # Currently booth Exes ar build with /subsystem:windows,5.02
```

Comment:
typos: "Currently both executables are built with /subsystem:windows and the console is attached manually"